css/components/sidebar.css

```diff
@@ -21,22 +21,23 @@
   display: block;
   cursor: pointer;
   transition: all 0.2s linear;
-  border-radius: 0.25rem;
```

Comment:
is there any reason to use rem and not px as it is less than 1 rem

Reply:
I calculate everything down to rem even if it px except when it is 1 or 2